splitapiclient/resources/segment_definition.py

```diff
@@ -172,3 +184,175 @@ def submit_change_request(self, keys, operation_type, title, comment, approvers,
             data['rolloutStatus'] = {'id': rollout_status_id}
         imc = require_client('ChangeRequest', self._client, apiclient)
         return imc.submit_change_request(self._environment['id'], workspace_id, data)
+
+    def get_keys_from_sdk_endpoint(self, sdk_api_key):
```

Comment:
If it's not too late, maybe `get_all_keys` is a better name.  (old one is get_keys)

Reply:
will do